lib/run/index.js

```diff
@@ -82,7 +82,7 @@ module.exports = function (options, callback) {
         runner.run(options.collection, {
             stopOnFailure: options.bail, // LOL, you just got trolled ¯\_(ツ)_/¯
             abortOnFailure: options.abortOnFailure,
-            iterationCount: options.iterationCount || _.get(options, 'iterationData.length') || 1,
+            iterationCount: _.get(options, 'iterationData.length') || options.iterationCount || 1,
```

Comment:
If iteration count is specified, it should override the length of iteration data.